### PR TITLE
Added login method to FBPageTest

### PR DESCRIPTION
**Pull request by thedreamskater**
Hi DreamCoder,
Added login method to FBPageTest. Please review.

Regards
TheDreamSkater

### CODE REVIEW

File: src/test/java/baseClasses/FBPageTest.java [hunk 1]
```diff
@@ -23,6 +26,15 @@ public void demoTest()
 		driver.close();
 	}
 	
+	
+	public void login()
+	{
+		driver.findElement(By.xpath("//input[@name='email']")).sendKeys("amar.genius@yahoo.com");
+		driver.findElement(By.xpath("//input[@name='pass']")).sendKeys("Goalreminder12!");
+		driver.findElement(By.xpath("//button[@name='login']")).click();
+
```
Comment (dreamcoder009) [inline src/test/java/baseClasses/FBPageTest.java]: add driver.close() here

File: src/test/java/baseClasses/FBPageTest.java [hunk 2]
```diff
@@ -23,6 +26,15 @@ public void demoTest()
 		driver.close();
```
Comment (thedreamskater) [inline src/test/java/baseClasses/FBPageTest.java:26]: remove this driver.close()

File: src/test/java/baseClasses/FBPageTest.java [hunk 3]
```diff
@@ -23,6 +26,15 @@ public void demoTest()
 		driver.close();
 	}
 	
+
```
Comment (thedreamskater) [inline src/test/java/baseClasses/FBPageTest.java:29]: Add this test case to the Test annotation and add to your testing.xml file as well.

File: src/test/java/baseClasses/FBPageTest.java (hunk 2 above)
Comment (dreamcoder009) [inline src/test/java/baseClasses/FBPageTest.java]: remove this driver.close() as we are closing it in the login() method

File: src/test/java/baseClasses/FBPageTest.java (hunk 3 above)
Comment (dreamcoder009) [inline src/test/java/baseClasses/FBPageTest.java:29]: Please add @Test testng annotation and add this method to the testing.xml file as well

File: src/test/java/baseClasses/FBPageTest.java (hunk 2 above)
Comment (thedreamskater) [reply src/test/java/baseClasses/FBPageTest.java]: Sure, I'll do that and revert back asap.

File: src/test/java/baseClasses/FBPageTest.java (hunk 3 above)
Comment (thedreamskater) [reply src/test/java/baseClasses/FBPageTest.java:29]: Sure, I'll do that and revert back asap.

File: src/test/java/baseClasses/FBPageTest.java (hunk 2 above)
Comment (thedreamskater) [reply src/test/java/baseClasses/FBPageTest.java]: Changes Done.

File: src/test/java/baseClasses/FBPageTest.java (hunk 3 above)
Comment (thedreamskater) [reply src/test/java/baseClasses/FBPageTest.java:29]: Changes Done.

**Review by dreamcoder009 — APPROVED** [review]
Looks Good, Please go ahead and merge the changes to the master branch.

**Pull request merged**